integreat_cms/cms/models/poi_categories/poi_category.py

```diff
@@ -1,14 +1,34 @@
+from django.db import models
 from django.utils.functional import cached_property
 from django.utils.translation import get_language, gettext_lazy as _
 
 from ..abstract_base_model import AbstractBaseModel
+from ..media.media_file import MediaFile
+from ...constants import poicategory
 
 
 class POICategory(AbstractBaseModel):
     """
     Data model representing a POI category.
     """
 
+    icon = models.ForeignKey(
+        MediaFile,
+        verbose_name=_("icon"),
+        on_delete=models.SET_NULL,
+        blank=True,
+        null=True,
+    )
```

Comment:
As described in https://github.com/digitalfabrik/integreat-cms/issues/1964#issuecomment-1351780506, the icon should not be a dynamic media file, but also a charfield which contains the icon name with a fixed set of choices.

All icons which can be selected can be downloaded here and should be added to the static files of this repository:
https://nextcloud.tuerantuer.org/index.php/apps/files/?dir=%2FT%C3%BCr+an+T%C3%BCr+-+Digitalfabrik%2FB+11.+Tech%2FPoi+Icons&fileid=3345504